sqlx-sqlite/src/connection/serialized_database.rs

```diff
@@ -0,0 +1,32 @@
+#![allow(unsafe_code)]
+extern crate libsqlite3_sys as ffi;
```

Comment:
This doesn't follow the import convention we've used in the rest of the project. You can just import `sqlite3_free()` directly, the `sqlite3` prefix should be sufficient to signal that it is an FFI call to anyone who is familiar with the SQLite C API.